src/Catalog/Helpers/Utils.cs

```diff
@@ -123,25 +221,12 @@ public static ZipArchive GetPackage(Stream stream)
             return package;
         }
 
-        public static XDocument NormalizeNuspecNamespace(XDocument original)
-        {
-            XDocument result = new XDocument();
-
-            using (XmlWriter writer = result.CreateWriter())
-            {
-                XslCompiledTransform xslt = new XslCompiledTransform();
-                xslt.Load(XmlReader.Create(new StreamReader(GetResourceStream("xslt.normalizeNuspecNamespace.xslt"))));
-                xslt.Transform(original.CreateReader(), writer);
-            }
-
-            return result;
-        }
-
+        // revisit to remove dead code (including view.xslt, view.html)? - skip caching for now
         public static string CreateHtmlView(Uri resource, string frame, string baseAddress)
```

Comment:
It's dead code, and wasn't originally cached. Original cache was concurrent dictionary on CatalogContext, only used for nuspec and normalizeNamespace transforms.

Will go ahead and move the dead code in this change.